app/javascript/components/upload/ProcessedExpressionStep.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default {
+  title: 'Processed Expression',
+  name: 'processedExpression',
```

Comment:
Could we wordsmith more parallel phrases for "processed expression" and "raw counts"?  They're tightly coupled, and phrases that cohere better would make it easy for users to connect them in their minds.

Straw proposals:
* Processed counts (and raw counts)
* Raw expression (and processed expression)